modules/distributing/slides-distributing.md

```diff
@@ -108,4 +108,4 @@ Specify the dependencies in a file and let the user build their own environment,
 - If other software might depend on this software, package it
 - To archive a specific software version and its environment, you could use a container
 
-===
\ No newline at end of file
+===
```

Comment:
```suggestion
```

I don't think we want this slide separator here as the Rules of thumb is the last slide at the moment.